### ISSUE TITLE
I found that removing the prk configuration, the video bitrate can reach 12k

### ISSUE THREAD

**Issue by lkmvip** [lkmvip/netflix-4K-DDplus#67]
I found that removing the prk configuration, the video bitrate can reach 12k

But Netflix provides some programs with a bitrate exceeding 12K, for example, the bitrate of movieid:81021057 is 17K

So in the future I will add an option for those who need it

use prk config
![111111](https://github.com/lkmvip/netflix-4K-DDplus/assets/12974141/0228304e-34d5-42f5-bdef-37fb51ba44de)

do not use
![222222](https://github.com/lkmvip/netflix-4K-DDplus/assets/12974141/932b7ac7-354e-47b3-93d4-4809671b3ba1)

**Comment by Jowat97** [comment]
@lkmvip Very amazing! So you can switch between latter bitarte and dynamic bitrate as far as I understood? Nice!
But I think regular latter Bitrate is up to 16.000? Anyways, very cool! 😃 
Is there a way I can test it already?

**Comment by lkmvip** [comment]
the current way to implement mandatory 12K is to fill in the missing config and remove prkconfig, inspired by Netflix’s UWP APP, but Netflix does not seem to provide a 16K solution for SDR

**Comment by Jowat97** [comment]
Okay, I understand. Yeah, I also remember they removed 16K during covid. Sad, that they didn't brought it back.
Would you say the 12K (fixed ladder) is a better quality than optimized bitrate ladder? According to their blog entry, they say optimized bitrate ladder should be better: https://netflixtechblog.com/optimized-shot-based-encodes-for-4k-now-streaming-47b516b10bbb
But I'm really not sure about that..
However, will this function be implemented in the extension, which can be activated? Would be amazing! 😃

**Comment by lkmvip** [comment]
In theory, a high bit rate is better than a low bit rate. Although Netflix has repeatedly emphasized that VMAF will not reduce quality, it is still providing SDR 12K bitrate for its UWP APP.So I personally can't accept Netflix's explanation about VMAF.

Especially as shown in the screenshot above, the bitrate of a large number of 4K episodes and movies is lower than 8000K. Only Netflix can benefit from this, because they only need to pay for lower network bandwidth (network bandwidth fee Usually more expensive, for example local network operators in South Korea require Netflix to pay extra for network bandwidth)

So, since the UWP APP can use 12K bitrate, why can’t we also use it in EDGE? As mentioned above, using the 12K solution will lose some movies and episodes with a bitrate exceeding 12K. So should we use it?  I'll add an option and leave the choice to you.

I will release this feature in 2 days, there are still some details to be worked out

**Comment by Jowat97** [comment]
Thanks for clarification. You are absolutely right. They increase the prices but meanwhile decrease the quality..
I'm from europe, as they decreased the bitrate in europe during covid, I was able to get 16K by using USA - New York VPN. Not sure if this still works, I don't think so. But was also confusing, why they limited bitrate in europe and asia, but not in the USA.
I have a 4K HDR Vision TV too, I noticed that the HDR content on the native TV app sometimes has a **way** higher bitrate, than SDR content. I think 17.4 Mbps was the highest I noticed.  The SDR bitrate on the same movie was something like ~7.3K or so. Unfortunately I can't remember the movie anymore.

And yeah, I would use it. I would check if a movie or episode has a higher bitrate with UWP method or normal method and then turn the option on or not. ☺️
